.github/workflows/preview.yml

```diff
@@ -0,0 +1,53 @@
+name: Publish Preview release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish:
+    name: Publish Preview to VS Code Marketplace
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Set preview in package.json
+        id: set-preview
+        run: |
+          ./build/preview
+
+      - name: Read extension version
+        id: ext-version
+        run: |
+          content=`cat ./package.json | jq -r .version`
+          echo "::set-output name=content::$content"
+
+      - name: Read Node.js version
+        id: nodejs-version
+        run: |
+          content=`cat ./.nvmrc`
+          echo "::set-output name=content::$content"
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          # See https://github.com/actions/setup-node/issues/32
+          node-version: ${{ steps.nodejs-version.outputs.content }}
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: vsce package
+        run: npx -- vsce package --out=${{ runner.temp }}/terraform-${{ steps.ext-version.outputs.content }}.vsix
+
+      - name: Check latest published version
```

Comment:
Do we need this step?

Reply:
Technically no, but it is useful to verify in the logs of the action that the version was properly set. This may be premature, but I'm thinking of a case where the magic `build/preview` script does not update the version correctly and we need to see what was done.